source/extensions/filters/http/dynamo/dynamo_request_parser.cc

```diff
@@ -38,6 +38,7 @@ const std::vector<std::string> RequestParser::SUPPORTED_ERROR_TYPES{
     // 4xx
     "AccessDeniedException",
     "ConditionalCheckFailedException",
+    "IdempotentParameterMismatchException",
```

Comment:
Can we add tests for the failure scenarios that lead to these type of errors?

Reply:
Judging from the implementation of the function that uses this list, its just checking if this error name exists in the error string returned from dynamo. And the only tests for that function are passing a string with one of the error names in it, rather than actually mimicking the error itself. So is this necessary to add?